cmd/receptor-cl/receptor.go

```diff
@@ -19,9 +19,10 @@ import (
 )
 
 type nodeCfg struct {
-	ID            string                       `description:"Node ID. Defaults to local hostname." barevalue:"yes"`
-	DataDir       string                       `description:"Directory in which to store node data"`
-	FirewallRules []netceptor.FirewallRuleData `description:"Firewall Rules (see documentation for syntax)"`
+	ID                       string                       `description:"Node ID. Defaults to local hostname." barevalue:"yes"`
+	DataDir                  string                       `description:"Directory in which to store node data"`
+	FirewallRules            []netceptor.FirewallRuleData `description:"Firewall Rules (see documentation for syntax)"`
+	MaxIdleConnectionTimeout string                       `description:"User defined maximum time a backend connection can go without data before we consider it failed." default:"21s"`
```

Comment:
let's not include a default here, if not defined in the config it will just be an empty string. That way we still only have one place where we define the default, the constant at the top of the netceptor.go file

Reply:
if we remove this default, SetMaxConnectionIdleTime should return if given an empty string

Reply:
@fosterseth - ok good suggestion. not including a default here will resolve the previous code comment you made :)